src/NiSparseArrays.jl

```diff
@@ -1,5 +1,6 @@
 module NiSparseArrays
 
+using LinearAlgebra: eltype
```

Comment:
This is just the already exported `eltype` so no need to add this line.

Reply:
> using LinearAlgebra: eltype
It was added by VSCode and sometimes I just forgot to delete it...
> .vscode/launch.json removed
Maybe I could add filter in .gitignore file?

Reply:
Yes it makes sense to ignore the entire .vscode folder in gitignore